Adjust security configuration for new External API endpoints

staubesv opened the pull request:
H @krinnewitz, this adds the missing adjustments to the security configuration and the missing ROLE_API_* roles. This is required if the External API user does not have ROLE_ADMIN (which is usually the case)

staubesv committed:
Adjust security configuration for new External API endpoints

krinnewitz commented:
Hi @staubesv, thank you. Shouldn't we also list the new roles in `docs/guides/admin/docs/configuration/external-api.md` ?

staubesv commented:
I'll update the PR

staubesv committed:
Document new External API roles

Added documentation for the new External API roles ROLE_API_EVENTS_SCHEDULING_EDIT, ROLE_API_EVENTS_SCHEDULING_VIEW and ROLE_API_CAPTURE_AGENTS_VIEW

staubesv commented:
Done.
